platforms/forge/src/main/java/com/pg85/otg/forge/commands/runnables/MapWriter.java

```diff
@@ -0,0 +1,233 @@
+package com.pg85.otg.forge.commands.runnables;
```

Comment:
Elegant solution! If this is executed asynchronously though, it may access shared resources in an unsafe way, which could cause problems f.e. if the world is unloaded while the MapWriter is running?